setup.py

```diff
@@ -39,7 +39,7 @@
       scripts=glob('scripts/*py'),
       setup_requires=['numpy >= 1.9.2'],
       install_requires=[
-          'IPython < 4.0.0',
+          'IPython',
```

Comment:
This is an issue of personal preference, not functionality. I don't like opening the markdown file, editing it, and then opening the notebook with IPython. I can, and the package works fine. (Actually, it only works that way with IPython 4.0, because there's a backwards compatibility changed between 3.0 and 4.0 in where the file is changed.) But, I'd rather operate in the ipynb framework and then do the conversion.